anap1003/assignment3/Assignment3.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package assignment3;
+
+import java.util.BitSet;
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+/**
+ *
+ */
+public class Assignment3 {
+    private int rows, cols;

```

Comment:
It is worth remembering for the future that having rows,cols and foundWords as member variables means that you cannot run wordSearch in multiple threads at the same time. Generally this isn't a problem, but if you start writing servers that handle multiple user inputs at the same time it can become an issue.